turbomoleio/testfiles/utils.py

```diff
@@ -649,7 +650,8 @@ def run_itest(executables, define_options, coord_filename, control_reference_fil
             try:
                 ret_code = process.wait()
 
-                if ret_code or "ended normally" not in program_std_err:
+                # The riper executable does not echo "ended normally" in the std err file...
+                if ret_code or (executable != "riper" and "ended normally" not in program_std_err):
```

Comment:
so there is no way to know from the output if a riper calculation finished correctly? :(

Reply:
Not from the stderr file unfortunately ...

Reply:
🤦